tests/bluetooth/hci_prop_evt/prj.conf

```diff
@@ -3,7 +3,7 @@ CONFIG_ZTEST=y
 
 CONFIG_BT=y
 CONFIG_BT_CTLR=n
-CONFIG_BT_NO_DRIVER=y
+CONFIG_BT_H4=n
```

Comment:
Given how many places you'd had to change `CONFIG_BT_NO_DRIVER=y` to `CONFIG_BT_H4=n` makes me wonder whether we should have kept `CONFIG_BT_NO_DRIVER`

Reply:
> Given how many places you'd had to change `CONFIG_BT_NO_DRIVER=y` to `CONFIG_BT_H4=n` makes me wonder whether we should have kept `CONFIG_BT_NO_DRIVER`

`BT_NO_DRIVER` relates to the fact that previously all drivers were behind a `choice`, meaning selecting one would deselect all others. As such it doesn't make sense to reintroduce it. With the new model, disabling these is primarily related to avoiding building in unused code. You can have as many drivers as you like, but the current host and hci_raw implementations only support a single instance.

Reply:
My point was that `CONFIG_BT_NO_DRIVER=y` was more obvious than `CONFIG_BT_H4=n` in terms of what we want to achieve here. 

If we keep it like this, perhaps we can simply add a comment stating that we are disabling the only driver that is enabled by default?